RustFxA/RustFirefoxAccounts.swift

```diff
@@ -18,11 +18,11 @@ final class Unknown: NSObject, NSCoding {
     }
 }
 
+// TODO: renamed FirefoxAccounts.swift once the old code is removed fully.
```

Comment:
We can probably remove that todo? Otherwise can't really say what this "old code" is anyway, and the file was never renamed, so looks like this is its permanent name (since that comment was made in 2020)

Reply:
I'll remove it in a separate PR to keep things moving. :)